README.md

```diff
@@ -262,15 +301,116 @@ datalad siblings remove -s ginhemio-storage
+If we expect to have large CSV files, we need to ensure that the CSV files are stored in the git-annex
+by adding this line to the `.gitattributes` file, which should be in the root directory of the repository:
+
+`*.csv annex.largefiles=anything`
+
+If the files we want to push to the R2 bucket are excluded by the `.gitignore` file, we need to
+remove the lines that exclude them.
+
+Now, we'll add R2 as a sibling (i.e. publication target) to the existing datalad repository.
+We have to use git-annex directly to do this.
+
+If you add a bucket with **public access**, use:
+```shell
+primap_datalad_creds  # skip if you set up your .bashrc to always inject the secrets
+git annex initremote public-r2 type=S3 encryption=none signature=v4 region=auto protocol=https \
+    autoenable=true \
+    bucket=primap-datalad-$name \
+    host=2aa5172b2bba093c516027d6fa13cdc8.r2.cloudflarestorage.com \
+    publicurl=$publicurl
+```
+If you add a bucket with **private access**, use:
+
+```shell
+primap_datalad_creds  # skip if you set up your .bashrc to always inject the secrets
+git annex initremote r2 type=S3 encryption=none signature=v4 region=auto protocol=https \
+    autoenable=true \
+    bucket=primap-datalad-$name \
+    host=2aa5172b2bba093c516027d6fa13cdc8.r2.cloudflarestorage.com
+```
+
+where you replace `$name` by the dataset name like `unfccc-di` and `$publicurl` by the
+public URL you copied when creating the cloudflare R2 bucket. If you didn't copy it,
+you can find it on the bucket's page in the settings under the heading "Public Access"
+in the section "R2.dev subdomain" at the entry "Public R2.dev Bucket URL". Copy it fully,
+it should look something like `https://pub-lotsofcharacters.r2.dev`.
+
+Now, the output of `datalad siblings` should look like this:
+```
+.: here(+) [git]
+.: public-r2(+) [git]
+.: origin(-) [https://github.com/mikapfl/unfccc_di_data.git (git)]
+```
+Note that your github sibling might not be named "origin" and your r2 remote will be called "public-r2" or "r2".  
+
+Now, we'll add a publication dependency on r2 to the github remote:
+```shell
+datalad siblings configure -s $github_sibling_name --publish-depends $r2_sibling_name
+```
+Replace `$github_sibling_name` with the name of your github sibling (usually, `github`
+or `origin`). Replace `$r2_name` with the name of the R2 sibling (`public-r2` or `r2`).
+
+**Option A: If we are confident that all the files are stored on disc**,
+we can **push** the dataset to github, which will automatically push to R2 as well,
+because we configured it as a publication dependency. This might take a while because
+it transfers all data:
+
+```shell
+datalad push --to $github_sibling_name
+```
+**Option B: If some files are only on a remote** (they are symlinks on our local machine),
+we could either download and the upload again
+(`datalad get .` and `datalad push --to $github_sibling_name`) or we can use git-annex 
+directly to **copy** the files (faster for large data sets):
```

Comment:
This is not possible if we are converting a pure git repo. There are no other remotes, so only Option A is possible.